### ISSUE TITLE
Params.json example

### ISSUE THREAD

**Issue by vincent-leguen** [hihihihiwsf/AST#3]
Hi @hihihihiwsf ! Congratulations for your very interesting paper and thank you for sharing the code ! I am trying to run your code, could you please provide an example of your params.json file ?

**Comment by qishibo** [comment]
+1

**Comment by razvanc92** [comment]
I managed to create a json file which works. Here is the example I come up with:

{
  "d_model": 160,
  "h": 8,
  "attn_type": "entmax15",
  "d_ff": 16,
  "dropout": 0.3,
  "num_class": 2000,
  "embedding_dim": 20,
  "cov_dim": 16,
  "N": 3,
  "predict_steps": 24,
  "train_window": 192,
  "batch_size": 64,
  "predict_batch": 64,
  "num_epochs": 500,
  "lr_d": 0.0001,
  "lr": 6.25e-5,
  "lr_schedule": "warmup_cosine",
  "lr_warmup": 0.002,
  "predict_start": 168,
  "gan": true,
  "test_predict_start": 168
}

it must be added under: experiments\test\parameters.json. Most of the attributes are self explanatory, except for num_class, which I couldn't figure it out yet. If you're noticing any difference from the paper please let me know, so I can modify the example.

**Comment by af258963** [comment]
> I managed to create a json file which works. Here is the example I come up with:
> 
> {
> "d_model": 160,
> "h": 8,
> "attn_type": "entmax15",
> "d_ff": 16,
> "dropout": 0.3,
> "num_class": 2000,
> "embedding_dim": 20,
> "cov_dim": 16,
> "N": 3,
> "predict_steps": 24,
> "train_window": 192,
> "batch_size": 64,
> "predict_batch": 64,
> "num_epochs": 500,
> "lr_d": 0.0001,
> "lr": 6.25e-5,
> "lr_schedule": "warmup_cosine",
> "lr_warmup": 0.002,
> "predict_start": 168,
> "gan": true,
> "test_predict_start": 168
> }
> 
> it must be added under: experiments\test\parameters.json. Most of the attributes are self explanatory, except for num_class, which I couldn't figure it out yet. If you're noticing any difference from the paper please let me know, so I can modify the example.

@razvanc92 thanks for share the json,I tried to run the training process, but I am stuck in many Ipython sections
